english/watch-star-wars/WWIII.md

```diff
@@ -0,0 +1 @@
+After the wall is completed, President Trump sends Mexico a bill for $8 billion. Mexico doesn't like that, and declares war on the U.S. The rest is downhill from here...
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.